domain-services/fiscal-event-service/src/main/java/org/egov/util/MasterDataConstants.java

```diff
@@ -0,0 +1,30 @@
+package org.egov.util;
+
+public class MasterDataConstants {
+    public static final String PROJECT_ID = "PROJECT_ID";
+    public static final String JSONPATH_ERROR = "JSONPATH_ERROR";
+    public static final String FISCAL_EVENT = "FISCAL_EVENT";
+    public static final String REFERENCE_ID = "REFERENCE_ID";
+    public static final String EVENT_TYPE = "EVENT_TYPE";
+
+    private MasterDataConstants() {
```

Comment:
Remove unused constructor

Reply:
It is not unused, it is created to prohibit new instance creation while this class is only meant for public static final variable.